bin/merge_taxonomy.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
+
+# Author: @darcy220606
```

Comment:
Can you also add to all the custom scripts text akin to: https://github.com/nf-core/mag/blob/e486bb23b9012ecf649061f1eb448bb853e0aad8/bin/domain_classification.R#L3-L4